data/service/service/standard.go

```diff
@@ -273,3 +276,30 @@ func (s *Standard) initializeServer() error {
 
 	return nil
 }
+
+func getenvStr(key string) (string, error) {
+	ErrEnvVarEmpty := errors.New("getenv: environment variable empty")
+	v := os.Getenv(key)
+	if v == "" {
+		logrusLogger.Debug("environment variable empty")
+		return v, ErrEnvVarEmpty
+	}
+	return v, nil
+}
+
+// Retrieve the PUSH_TO_READ_STORE_ENABLED env variable
+// It accepts 1, t, T, TRUE, true, True, 0, f, F, FALSE, false, False.
+// Any other value returns true by default.
+func getPushToReadStoreEnv(key string) bool {
```

Comment:
why passing the key as a variable if the function is named after the key? It will be used for this key only.
Even the messages bellow are refering to the env var name (hardcoded)

I would remove the key var

Reply:
good point i will remove it